financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/settings/EmailDefinition.kt

```diff
@@ -0,0 +1,32 @@
+package com.stripe.android.financialconnections.example.settings
+
+import com.stripe.android.financialconnections.example.data.LinkAccountSessionBody
+import com.stripe.android.financialconnections.example.data.PaymentIntentBody
+import com.stripe.android.financialconnections.example.settings.PlaygroundSettingDefinition.Displayable.Option
+
+internal object EmailDefinition : PlaygroundSettingDefinition.Displayable<String>,
+    PlaygroundSettingDefinition.Saveable<String> {
+    override val displayName: String = "Customer email"
+    override val options: List<Option<String>> = emptyList()
+
+    override fun lasRequest(
+        body: LinkAccountSessionBody,
+        value: Any?
```

Comment:
How come this isn't typed?

Reply:
my bad - https://github.com/stripe/stripe-android/pull/7448/commits/5dcf82f26b13e29bb2fa868c5d5f5df464960970